### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by tugsem**
In the homepage I: 

1. added index page and a title.
2. added some color to the title.
3. added required files.(test.md, .gitignore, html-css linters)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
+2. Use `npx stylelint "**/*.scss"` to lint you SCSS files and not the generated CSS
+
+Once you've done the previous steps you can focus on your SCSS files and not in fixing the generated CSS output
+every time.
```
Comment (ShadyShawkat) [inline README.md]: - [ ] Kindly use [This README template](https://github.com/microverseinc/readme-template) provided by Microverse, and please make sure to edit each section with the proper information according to your project.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @tugsem, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## To Highlight ✔️ 🥇 
- You used GitHub Flow correctly.
- Your pull request has a descriptive title and a meaningful description.
- Good job giving commits descriptive messages.
- Linters are set up correctly and all checks are green.
- Nice work ignoring the `test.md` file in your `.gitignore`.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+- HTML, CSS, Node
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@tugsem]
+- Twitter: [@TugsemSaranli]
+- LinkedIn: [Tugsem Saranli]
```
Comment (keroloussamy) [inline README.md]: - [x] Kindly add the links to your accounts. So users can reach you.
like 
- GitHub: `[tugsem](HTTP://................)`

Comment (keroloussamy) [inline README.md]: - [x] Kindly add the rest of the sections from [this template](https://github.com/microverseinc/readme-template) on your readme.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @tugsem, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (tugsem) [reply README.md]: @keroloussamy Thanks for feedback. I have rearranged readme file.

**Review by prolajumokeoni — APPROVED** [review]
Hi @tugsem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**